src/library_syscall.js

```diff
@@ -157,11 +157,6 @@ var SyscallsLibrary = {
   },
 
   _munmap_js__i53abi: true,
-  _munmap_js__deps: ['$SYSCALLS',
-#if FILESYSTEM && SYSCALLS_REQUIRE_FILESYSTEM
-    '$FS',
-#endif
-  ],
```

Comment:
What about line 170 below which uses `SYSCALLS`?

Reply:
All function that are considers "syscalls" (i.e. all function in this file and in library_wasi.js) all get the `SYSCALL` dependency implicitly added by `wrapSyscallFunction` so there is no need to add in explicitly.  Same goes for `FS`